src/main/kotlin/graphics/scenery/controls/TrackedStereoGlasses.kt

```diff
@@ -291,6 +291,10 @@ class TrackedStereoGlasses(var address: String = "device@localhost:5500", var sc
         return shift
     }
 
+    override fun fadeToColor(seconds: Float, color: Vector3f, alpha: Float, background: Boolean) {
+        TODO("Not yet implemented")
```

Comment:
Just as a note, no need to do this right now - this could be easily implemented with a cam-attached plane that fades, and is removed after the fade 👍